docs/user/getting_started.rst

```diff
@@ -155,14 +155,16 @@ usually be set on the command line or in ``site.mk``.
 Common variables
 ................
 
-GLUON_ATH10K_MESH
+GLUON_WLAN_MESH
   While Gluon does support some hardware with ath10k-based 5GHz WLAN, these WLAN adapters don't work
   well for meshing at the moment, so building images for these models is disabled by default. In addition,
   ath10k can't support IBSS and 11s meshing in the same image due to WLAN firmware restrictions.
 
-  Setting GLUON_ATH10K_MESH to ``11s`` or ``ibss`` will enable generation of images for ath10k devices
+  Setting GLUON_MESH_MESH to ``11s`` or ``ibss`` will enable generation of images for ath10k devices
   and install the firmware for the corresponding WLAN mode.
 
+  Setting GLUON_MESH_MESH to ``11s`` will aso list targets their able to run only AP + 11s.
```

Comment:
I think this text should be a little more generic now. Maybe removing the references to ath10k is sufficient. Or as an alternative, you could add other examples ralink/mediatek...

Reply:
i agree, the text should reflect the other targets as well, removing the text about ath10k is not necessary, as that content still applies. (this and lemoers comment affect this whole paragraph, not only the last line)
also, this last line has some english spelling/grammar issues. additionally, i'm not sure what "list" should mean in this context.

Reply:
I change the text above also a bit, because the part about the default state was not correct.